docs/learn/master-data-hub/index.md

```diff
@@ -60,9 +60,12 @@ Once the extension is activated for a queue, it will be automatically triggered
 
 ![Master Data Hub: select queue](./img/mdh-select-queue.png)
 
-### Step 4: Assign the token owner
+### Step 4: Assign the token owner (optional)
 
-Token owner must have an "Admin" role assigned in Rossum. User queries the database and adds the results to annotation data.
+For our clients, this step is not necessary. An account with admin rights will be able to create the extension, and the owner of the extension will automatically be assigned to the account that created it. Therefore, this step is not displayed in the User Interface.   
```

Comment:
I wonder whether this is a good approach or if we should keep recommending the system user anyway. I feel like having a separate account for the integrations is much better approach. 🤔